docs/keys.md

```diff
@@ -1,4 +1,6 @@
-# Keys
+# Keys subcommand
```

Comment:
```suggestion
# Keys management
```

Reply:
i like it. what do you think of `Key management`?

Reply:
Sounds good :rocket:

Reply:
Sounds good :rocket: